src/sentry/attachments/redis.py

```diff
@@ -8,11 +8,28 @@
 
 from __future__ import absolute_import
 
+import logging
+
+from django.conf import settings
+
 from sentry.cache.redis import RedisCache
+from sentry.utils import redis
 
 from .base import BaseAttachmentCache
 
+logger = logging.getLogger(__name__)
+
 
 class RedisAttachmentCache(BaseAttachmentCache):
     def __init__(self, **options):
-        super(RedisAttachmentCache, self).__init__(RedisCache(**options), **options)
+        cluster_id = getattr(
+            settings,
+            'SENTRY_ATTACHMENTS_REDIS_CLUSTER',
+            'attachments',
+        )
+        try:
+            cache = redis.redis_clusters.get(cluster_id)
+        except KeyError:
+            cache = RedisCache(**options)
```

Comment:
Updated.
@mattrobenolt Not sure about this fallback, though. Would it be better to fail instantly if we try to use this cache backend but cannot find the cluster?

Reply:
Yes, I agree. If you're explicitly configuring this backend, and you configure it wrong, it shouldn't gracefully fallback.

Reply:
Removed the try-except.